src/App.scss

```diff
@@ -20,7 +20,7 @@ button {
   border-radius: 0.2rem;
   font-size: 1.2rem;
   text-decoration: none;
-  color: $base_white;
+  color: #ffffff;
```

Comment:
Why not modify $base_white color ?

Reply:
I just wanted to change color in button to have enough contrast in them, not to modify background or other elements

Reply:
Change $base_white so have fix constrat every where

Reply:
Done